File: Player.cpp
/* Honor Pledge:
*
* I pledge that I have neither given nor 
* received any help on this assignment.
*
* Youngjon
*/ 

#include <iostream>
#include <string>
#include <stdlib.h>
#include "Player.h"

using namespace std;

void Player::set_firstname( string name, int jerseyNum){
	firstName[jerseyNum] = name;
}
void Player::set_lastname( string name, int jerseyNum){
	lastName[jerseyNum]= name;
}
void Player::set_jerseynumbers(int jerseyNum){
	jerseyNumbers[jerseyNum] = 1;
}
int Player::check_slot( int jerseyNum){
	if( jerseyNumbers[jerseyNum] != 0){
		return 1;
	}
}
void Player::print_roster(){
	for( int i= 0; i<101; i++){
		if( jerseyNumbers[i] == 1){
			cout << i;
			cout << ")";
			cout <<firstName[i] +" "+ lastName[i] << endl;
			cout << endl;
		}
	}
}
string Player::return_firstname(){
	return firstName[101];
}

File: Player.java
// Honor Pledge:
//
// I pledge that I have neither given nor 
// received any help on this assignment.
//
// youngjon

public class Player{
	private String[] fName = new String[101];
	private String[] lName = new String[101];
	private int[] jerseyNum = new int[101];
	
	public Player(String fName, String lName, int numBer){
		if (this.jerseyNum[numBer] == 0){
			this.fName[numBer] = fName;
			this.lName[numBer] = lName;
			this.jerseyNum[numBer] = 1;
		}
	}
	
}